ee/clickhouse/views/insights.py

```diff
@@ -43,27 +38,9 @@ def trend(self, request: Request, *args: Any, **kwargs: Any) -> Response:
 
     @action(methods=["GET"], detail=False)
     def session(self, request: Request, *args: Any, **kwargs: Any) -> Response:
+        response = ClickhouseSessions().run(team=self.team, filter=Filter(request=request))
```

Comment:
This split feels much better haha. There were too many conflicting functions here